nix/pkgs/default.nix

```diff
@@ -99,7 +99,7 @@ let
 
   # not available in 20.03 and we depend on several recent changes
   # including stylish-haskell support
-  nix-pre-commit-hooks = import (sources."pre-commit-hooks.nix");
+  nix-pre-commit-hooks = (pkgs.callPackage ((sources."pre-commit-hooks.nix") + "/nix/default.nix") {}).packages;
```

Comment:
Does this work? The arguments to the `default.nix` file are `sources` and `system`: we don't have `sources` in `pkgs`, and I don't think there's a `system` attribute in there either! Normally you have to pass in a nixpkgs *at* the right system already, I think?

Reply:
no you are completely right it won't work - this PR should use a branch of `pre-commit-hooks.nix` with some changes for this to make any sense really..

Reply:
I think now it should make more sense. I am using https://github.com/gilligan/pre-commit-hooks.nix/tree/testing for this.